### PR TITLE
Catalog modifications

### PR DESCRIPTION
**Pull request by Joel-Cripps**
Requested changes for the IBM catalog single image.

Changes include:
 - No longer automatically creates vpc/subnet/ etc. All of those will need to be input by the end user and created ahead of time.
 - SSH key is now a data reference
 - TF_VERSION enforced for catalog use.
 - 7.0 is now the image default.

Will need to be tested and validated before release.

### CODE REVIEW

**Review by jamie-pate — APPROVED** [review]

**Review by JaydenLiang — APPROVED** [review]
looks good to me.

**Comment by Joel-Cripps** [comment]
Adjusted a couple of things:
1. Now uses just the standard 7.0 image. Map vars fail to parse in the validation.
2. New README updates using full links. Will now show in IBM if no links present.

File: README.md [hunk 1]
```diff
@@ -1,52 +1,63 @@
...
 -   A Floating Public IP address attached to the FortiGate
 -   A Logging disk
--   A basic bootstrap, including license if supplied.
 
-## Deployment
+## Deployment:
+
+1. Fill in the required VPC, Subnet and ssh key values as shown below:
+
+![IBM FortiGate Deploy](https://raw.githubusercontent.com/fortinet/ibm-fortigate-terraform-deploy/main/imgs/imgs/step_6_a.png)
```
Comment (jamie-pate) [inline README.md:25]: imgs/imgs ?
https://raw.githubusercontent.com/fortinet/ibm-fortigate-terraform-deploy/main/imgs/step_6_a.png seems correct

Comment (jamie-pate) [reply README.md:25]: also a bunch of cases below

**Review by jamie-pate — CHANGES_REQUESTED** [review]

**Review by jamie-pate — APPROVED** [review]

**Review by JaydenLiang — APPROVED** [review]

**Pull request merged**